src/teleop/gui/src/components/AutonWaypointEditor.vue

```diff
@@ -91,8 +91,8 @@
             @toggle="toggleTeleopMode($event)"
           />
         </div>
-        <div class="stuck-check">
-          <Checkbox name="Stuck" @toggle="roverStuck = !roverStuck"></Checkbox>
+        <div class="stuck-check" style="padding-inline: 25px">
+          <Checkbox name="Stuck" style="transform: scale(1.5)" @toggle="roverStuck = !roverStuck" ></Checkbox>
```

Comment:
Could add this transform to stuck-check as well specifically for Checkbox elements
i.e.
```css
.stuck-check Checkbox {
    transform: scale(1.5);
}
```